examples/README.md

```diff
@@ -0,0 +1,88 @@
+# Finch examples
+
+Examples included in Finch are scan path creation and various versions of a single line additive case.
+
+
+# Finch inputs
+
+All inputs are passed to Finch in a JSON format.
+
+## Temporal parameters (`time`)
+
+- `start_time`: Simulation start time
+  - units: `s`
+- `end_time`:	Simulation end time
+  - units: `s`
+- `Co`:	Courant number
+  - units: unitless
+- `total_output_steps`: Frequency to output grid information
+  - units: unitless
+- `total_monitor_steps`: Frequency to output timing information
+  - units: unitless
+
+## Spatial parameters (`space`)
+
+- `initial_temperature`: Initial temperature at all grid points
+  - units: `K`
+- `cell_size`: Grid cell size (per dimension)
+  - units: `m`
+- `global_low_corner`: Bottom corner of the physical domain
+  - units: `m`
+- `global_high_corner`: Top corner of the physical domain
+  - units: `m`
+- `ranks_per_dim`: MPI ranks per dimension (replaced if incompatible with resource set)
+  - units: unitless
+  - optional (defaults to MPI-determined cartesian domain decomposition)
+
+## Material properties (`properties`)
+
+- `density`: Material density
+  - units: `kg/m^3`
+- `specific_heat`: Specific heat capacity
+  - units: `J/g/K`
+- `thermal_conductivity`: Thermal conductivity
+  - units: `W/m/K`
+- `latent_heat`: Latent heat of fusion
+  - units: `J/kg`
+- `solidus`: Solidus temperature
+  - units: `K`
+- `liquidus`: Liquidus temperature
+  - units: `K`
+
+## Laser source parameters (`source`)
+- `absorption`: Laser absorption
+  - units: unitless
+- `two_sigma`: Beam size
+  - units: `m`
+- `scan_path_file`: File containing laser path information
+
+
+
+## Output sampling (`sampling`)
+This section is optional
+
+- `type`: Type of sampling
+  - options: `solidification_data` (outputs sampled solidification data with spatial position x, y, z; melting time tm; solidification time ts; solidification rate R; and (optionally) temperature gradients Gx, Gy, Gz)
```

Comment:
I'd call ts "solidification start time" (or the time at which the location goes below the liquidus temperature)